articles/protocols/oidc/index.md

```diff
@@ -5,47 +5,27 @@ description: What is the OpenID Connect protocol and how it works.
-
-To get an ID Token using `/authorize` you need to populate the `scope` request parameter with the value `openid` (`scope=openid`).
-
-For more information on ID tokens refer to [Auth0 id_token](/tokens/id_token).
-
-## Claims
-
-Each ID Token contains claims, which are statements about an entity (typically, the user) and additional metadata. A claim can be the user's name, email address, or roles.
+Let's use the example we mentioned earlier, signing into Auth0 using your Google account, for a high level overview on how the flow works:
 
-There are three types of claims: reserved, public, and private claims. For details you can refer to [JSON Web Tokens (JWT) in Auth0](/jwt#payload).
+1. When you choose to sign in to Auth0 using your Google account, Auth0 sends an **Authorization Request** to Google.
+1. Google authenticates your credentials or asks you to login if you are not already signed in, and asks for your authorization (lists all the permissions that Auth0 wants, for example read your email address, and asks you if you are ok with that).
+1. Once you authenticate and authorize the sign in, Google sends an ID Token, and usually an Access Token, back to Auth0.
+1. Auth0 can retrieve user information from the ID Token or use the Access Token to invoke a Google API.
 
-The [OpenID Connect specification](https://openid.net/specs/openid-connect-core-1_0.html) defines a set of standard claims, like `name`, `email`, `picture`, `address`, `updated_at`, and more. For the complete list refer to the [paragraph 5.1 of the OpenID Connect specification](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims).
+## ID Tokens
 
-In addition to these standard claims, other claims may be used as well.
+The [ID token](/tokens/id_token) is a [JSON Web Token (JWT)](/jwt) that contains identity data. It is consumed by the client and used to get user information like the user's name, email, and so forth, typically used for UI display. ID Tokens conforms to an industry standard (IETF [RFC 7519](https://tools.ietf.org/html/rfc7519)) and contain three parts: a header, a body and a signature.
 
-For example, we could add an extra property at our client's users using [User Metadata](/metadata). This property would be called `roles` and it would have either the value `ROLE_ADMIN` or the value `ROLE_USER`. The plan is to get this value upon user login and depending on the value customize the UI so the user can or cannot take admin actions.
+### Claims
 
-All we have to do now is specify this new metadata as part of the `scope` request parameter (`scope=openid roles`). Once we decode our ID Token the body payload will contain the property `roles` with the corresponding value for the logged in user.
+ID Tokens contain claims, which are statements (such as name or email address) about an entity (typically, the user) and additional metadata. There are three [types of claims](/jwt#payload): reserved, public, and private claims. 
```

Comment:
In this paragraph, we refer to two different categorizations about claims:
- reserved, public, or private
- standard or custom

This is all good but the way the content is confusing (we mention the types or claims, and then go on to talk about standard vs custom claims).

How do these two categorizations relate to one another? 
Are reserved claims the same as standard claims?
Are all public claims custom claims?
Can all custom claims be either public or private?
... and so on

Also, I followed the [types of claims](/jwt#payload) link and didn't understand a thing about what private claims are. Maybe you can push an update to that doc as part of this PR?